airflow/migrations/versions/e959f08ac86c_change_field_in_dagcode_to_mediumtext_.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Change field in DagCode to MEDIUMTEXT for MySql
+
+Revision ID: e959f08ac86c
+Revises: 64a7d6477aae
+Create Date: 2020-12-07 16:31:43.982353
+
+"""
+from alembic import context, op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = 'e959f08ac86c'
+down_revision = '64a7d6477aae'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():  # noqa: D103
+    if context.config.get_main_option('sqlalchemy.url').startswith('mysql'):
```

Comment:
```suggestion
    conn = op.get_bind()  # pylint: disable=no-member
    if conn.dialect.name == "mysql":
```
See: https://github.com/apache/airflow/blob/4fb312140fc15b46fa96e98ec0e3939d81109eb6/airflow/migrations/versions/61ec73d9401f_add_description_field_to_connection.py#L42

Reply:
I used this as a base:

https://github.com/apache/airflow/blob/4fb312140fc15b46fa96e98ec0e3939d81109eb6/airflow/migrations/versions/d2ae31099d61_increase_text_size_for_mysql.py#L37

And I think it's more "universal" in a way.

Reply:
(And I would rather tkeep consistency with the very similar change).

Reply:
I think it's consistent, unless we want to change past migration, which I doubt.

Reply:
The op.get_bind() style is used much more:

```
co~/code/airflow/airflow master*
airflow ❯ git grep conn.dialect.name ./airflow/migrations/versions | wc -l
54

~/code/airflow/airflow master*
airflow ❯ git grep context.config.get_main_option ./airflow/migrations/versions | wc -l
4
```

Reply:
OK. If there are no drawbacks, but we should change all of them not only this one.

Reply:
Just to be a more empathethic to anyone who in the future could choose a  non-preferred example as a start.

Reply:
Opened PR #12898